vnext/JSI/Shared/ChakraRuntime.h

```diff
@@ -248,15 +239,35 @@ class ChakraRuntime : public facebook::jsi::Runtime {
   ChakraObjectRef ToChakraObjectRef(const facebook::jsi::Value &value);
   std::vector<ChakraObjectRef> ToChakraObjectRefs(const facebook::jsi::Value *value, size_t count);
 
-  // Host function and host object helpers
+  // Convenience functions for property access.
+  ChakraObjectRef GetProperty(const ChakraObjectRef &obj, const ChakraObjectRef &id);
```

Comment:
>id [](start = 81, length = 2)

(nit) just curious, why in one function this is "id" and in another it's "name" shouldn't they be consistent?

Reply:
The difference is intentional as a JS property ID can either be backed by a Symbol or a name (a.k.a. string)

---
In reply to: [341261195](https://github.com/microsoft/react-native-windows/pull/3568#discussion_r341261195) [](ancestors = 341261195)